string data
float32 confidence
string answer
